sysutils/dmidecode/src/www/widgets/widgets/dmidecode.widget.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * Copyright (C) 2019 Smart-Soft Ltd.
```

Comment:
Shall I change this to "Smart-Soft" or every other "Smart-Soft" to "Smart-Soft Ltd." ?